examples/step-55/reference.py

```diff
@@ -3,35 +3,35 @@
 from sympy.physics.vector import ReferenceFrame, gradient, divergence
 from sympy.vector import CoordSysCartesian
 
-R = ReferenceFrame('R');
-x = R[0]; y = R[1];
+R = ReferenceFrame('R')
+x = R[0]; y = R[1]
```

Comment:
do we want new line here for consistency? (same below in couple of places with `;`)

Reply:
I can do that, but don't really care.

Reply:
See #7525.